### PR TITLE
Announcing the default model used by the pipeline (with a link).

### LINKED ISSUE #12845: Add Model Details for Pipeline API
# 🚀 Feature request

I think it will be great if we could add further details about what particular model the Pipeline API is using. In particular, we can add the following details relating to the models in a readable markdown file:
* Dataset used for training
* Type of transformer used
* Hyperparameters and neural net architecture
* Evaluation metrics: accuracy + precision + recall on test portion of dataset
## Motivation
I was working on different sentiment analysis datasets with different transformers when I stumbled across the HuggingFace pipeline API. I was impressed by the performance of the models, but I wasn't quantitively sure about the accuracy of the model, neither was I sure about how it was trained.

I believe that many other users of the Pipeline API will also benefit from such a change.

P.S. If there is some source of such information and I've missed it, please let me know

### PR DESCRIPTION
**Pull request by Narsil**
# What does this PR do?

Fixes https://github.com/huggingface/transformers/issues/12845

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM, thanks @Narsil!
